database/migrations/2015_09_14_135258_create_channels_table.php

```diff
@@ -26,6 +28,6 @@ public function up()
      */
     public function down()
     {
-        Schema::drop('password_resets');
+        //
```

Comment:
This function is empty. Are you not meant to drop the `channels` table in this function?